sdk/tables/Azure.Data.Tables/README.md

```diff
@@ -21,7 +21,7 @@ The Azure Tables client library can seamlessly target either Azure Table storage
 Install the Azure Tables client library for .NET with [NuGet][table_client_nuget_package]:
 
 ```
-dotnet add package Azure.Data.Tables --version 3.0.0-beta.3
+dotnet add package Azure.Data.Tables --version 3.0.0-beta.5
```

Comment:
I *think*  the `--prerelease` switch will automatically select the latest beta.

Reply:
Ah - yes, that is much better!